tools/data/sthv1/preparing_sthv1.md

```diff
@@ -7,16 +7,60 @@ Before we start, please make sure that the directory is located at `$MMACTION2/t
 
 First of all, you have to sign in and download annotations to `$MMACTION2/data/sthv1/annotations` on the official [website](https://20bn.com/datasets/something-something/v1).
 
-## Step 2. Prepare Videos
+## Step 2. Prepare RGB Frames or Videos
 
-Then, you can download all data parts to `$MMACTION2/data/sthv1/` and use the following command to extract.
+Since the sthv1 website doesn't provide the original video data and only extracted RGB frames are available, here are two alternatives to choose:
+a. For users who only want to use RGB frames, you can directly download RGB frames from [sthv1 website](https://20bn.com/datasets/something-something/v1).
+b. For users who need videos and RGB + Flow, you should download the videos from [sthv2 website](https://20bn.com/datasets/something-something/v2) and go with the following steps to extract them.
+
+### Plan A: Prepare RGB Frames
```

Comment:
What does it mean by "Plan A". Could you change it to a word that is less “cool”?

Reply:
How about "### a. prepare RGB frames" ?

Reply:
And note that  Capitalization only for first and second header.